rootfs/api/tests/test_app.py

```diff
@@ -283,6 +287,7 @@ def test_run(self, mock_requests):
         self.assertEqual(response.data['rc'], 0)
         self.assertEqual(response.data['output'], 'mock')
 
+    @mock.patch('scheduler.KubeHTTPClient._get_port', _mock_port)
```

Comment:
Could this decorator apply to the whole `AppTest` instead? I don't know if it would break other unit tests, but it would be less repetitive.